docs/source/devtools-overview.md

```diff
@@ -17,7 +17,7 @@ The ExecuTorch Developer Tools support the following features:
 - **Debugging** - Intermediate outputs and output quality analysis
 - **Numerical Discrepancy Detection** - Operator-level numerical discrepancy detection between AOT and runtime intermediate outputs to streamline numerical debugging and validation.
 - **Memory Allocation Insights** - Visualize how memory is planned, where all the live tensors are at any point in time
-- **Visualization** - Coming soon
+- **Visualization** - Visualize the model as a computational graph with highlighted partitions for easy debugging.
```

Comment:
```suggestion
- **Visualization** - Visualize the model as a computational graph
```
Note: As we are working on it, let provide also a brief generic documentation as missing so far. Not just our contribution.